### PR TITLE
Edited Swift doc

### PR DESCRIPTION
**Pull request by zardra**

### CODE REVIEW

File: docs/_docs/languages/swift.md [hunk 1]
```diff
@@ -19,79 +19,72 @@ Nuclide has *limited*, built-in support for [Swift](https://swift.org/) and the
 
 Nuclide will attempt to find a Swift executable automatically:
 
-- On macOS, it will use the latest version of Swift installed, at `/Library/Developer/Toolchains/swift-latest.xctoolchain`.
-- On Linux, it will find a Swift executable based on your PATH.
+- On macOS, it will use the latest version of Swift installed at `/Library/Developer/Toolchains/swift-latest.xctoolchain`.
+- On Linux, it will find a Swift executable based on your $PATH.
```
Comment (modocache) [inline docs/_docs/languages/swift.md]: Should we have `$PATH` be monospaced here?

File: docs/_docs/languages/swift.md [hunk 2]
```diff
@@ -19,79 +19,72 @@ Nuclide has *limited*, built-in support for [Swift](https://swift.org/) and the
...
-  `/usr/local/bin/sourcekitten`. Installing SourceKitten via Homebrew, by
-  running `brew install sourcekitten` at the command line, places it in this
-  location.
+Nuclide uses [SourceKitten](https://github.com/jpsim/SourceKitten) to provide features such as [Autocomplete]((#features__autocomplete)).
+
+On macOS, Nuclide expects to find a SourceKitten executable at `/usr/local/bin/sourcekitten`. Installing SourceKitten via [Homebrew](http://brew.sh/), by running `brew install sourcekitten` at the command line, places it in this location.
 
 > Unfortunately, SourceKitten is not yet available on Linux. As a result,
-  features such as autocompletion are only available on macOS.
+  features such as Autocomplete are only available on macOS.
```
Comment (modocache) [inline docs/_docs/languages/swift.md]: Should this be a link to the autocomplete section below?

Comment (JoelMarcey) [inline docs/_docs/languages/swift.md]: Good idea -- `[Autocomplete](#features__autocomplete)`

File: docs/_docs/languages/swift.md (hunk 1 above)
Comment (zardra) [inline docs/_docs/languages/swift.md]: Missed that one. Thanks!

**Comment by zardra** [comment]
Suggested changes made

**Comment by JoelMarcey** [comment]
landing manually.

**Pull request closed without merging**